module/request.py

```diff
@@ -31,21 +31,21 @@ def request(update: Update, context: CallbackContext) -> None:
         context.bot.send_message(chat_id, USERNAME_ERROR)
         return
 
-    user_isbn = message.split('; ')[0].split()[1]
+    user_isbn = message.split(';')[0].split()[1].replace(' ','')
     if len(user_isbn) != 10 and len(user_isbn) != 13:
         context.bot.send_message(chat_id, ISBN_ERROR)
         return
 
     try:
-        format(float(message.split('; ')[1].replace(",", ".")), ".2f")
-        price = str(format(float(message.split('; ')[1].replace(",", ".")), ".2f"))
+        format(float(message.split(';')[1].replace(",", ".").replace(" ","")), ".2f")
```

Comment:
is this line needed?
I was splitting the `price` value into multiple variable to improve the readability

Reply:
No, it wasn't. My mistake. I am going to correct this.